README.md

```diff
@@ -4,6 +4,19 @@ Tools to help you pick a prior.
 
 **This package is very new and all of its features are experimental, not yet well tested, and subject to change without notice.**
 
+## The Zen of PreliZ
+* Being open source. 
+* Avoid automated solutions.
+* Separate tasks among humans and computers.
+* Prevent overconfidence on users opinions.
+* Easily integrate with other tools.
+* Allow to perform prior predictive checks.
+* Having a simple and intuitive interface suitable for non-specialists in order to minimize cognitive biases and heuristics.
+* Switching between different types of visualization such as kernel density estimates plots, quantile dotplots, histograms, etc. 
+* Being agnostic of the underlying programming language, or at least interoperable with as many languages as possible.
```

Comment:
```suggestion
* Being agnostic of the underlying probabilistic programming language
```
This is a little bit mote easy to achieve, although we can make this not just a Python project but also a Julia project as with main ArviZ, but that is a little bit more complex